ui/components/app/assets/asset-list/cells/generic-asset-cell-layout.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { ReactNode, useState } from 'react';
+import { Box } from '../../../../component-library';
+import {
+  Display,
+  FlexDirection,
+  BlockSize,
+  JustifyContent,
+} from '../../../../../helpers/constants/design-system';
+
+type GenericAssetCellLayoutProps = {
+  onClick?: () => void;
+  disableHover?: boolean;
+  badge: ReactNode;
+  headerLeftDisplay: ReactNode;
+  headerRightDisplay: ReactNode;
+  footerLeftDisplay: ReactNode;
+  footerRightDisplay: ReactNode;
+};
+
+export default function GenericAssetCellLayout({
+  onClick,
+  disableHover = false,
+  badge,
+  headerLeftDisplay,
+  headerRightDisplay,
+  footerLeftDisplay,
+  footerRightDisplay,
+}: GenericAssetCellLayoutProps) {
+  const [isHovered, setIsHovered] = useState(false);
+
+  return (
+    <Box
+      display={Display.Flex}
+      flexDirection={FlexDirection.Row}
+      width={BlockSize.Full}
+      height={BlockSize.Full}
+      gap={4}
+    >
+      <Box
+        as="a"
+        onClick={(e?: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {
```

Comment:
Generic note for visibility. This was the original component behaviour, we can discuss with DS team if there is a better component we can use in future.